.github/workflows/avm.template.module.yml

```diff
@@ -25,6 +25,7 @@ env:
   ARM_MGMTGROUP_ID: "${{ secrets.ARM_MGMTGROUP_ID }}"
   ARM_TENANT_ID: "${{ secrets.ARM_TENANT_ID }}"
   TOKEN_NAMEPREFIX: "${{ secrets.TOKEN_NAMEPREFIX }}"
+  CI_KEY_VAULT_NAME: "${{ vars.CI_KEY_VAULT_NAME }}"
```

Comment:
Wondering if setting this as a secret would provide a few advantages, such as  hiding our value in the logs and make it slightly faster to set up for contributors (just set secrets and not both secrets and vars)
Just a though not a hard stopper

Reply:
Fair point. I did set it up explicitely as a variable because I'd not consider the value a secret - and because variables are in general less of a pain than secrets 😄 I guess it just made sense to then just use the GitHub feature that exists for that purpose. But if you say you'd like to hide the value I can refactor it.

Reply:
I'm good either way so will leave the final decision to you. I just wanted to raise the point in case we wanted to consider hiding the kv name in the logs